Database/src/main/resources/changelog1.xml

```diff
@@ -838,4 +838,23 @@
         </createTable>
         <addPrimaryKey tableName="tbl_tenant_config" columnNames="tenant_id, key" constraintName="tenant_specific_key" />
     </changeSet>
+    
+    <changeSet author="CAF" id="adding_repository_table">
+        <createTable tableName="tbl_repository_config">
+            <column name="tenant_id" type="VARCHAR(40)">
+                <constraints nullable="false"/>
+            </column>            
+            <column name="repository_id" type="VARCHAR(40)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="key" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="value" type="CLOB">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addPrimaryKey tableName="tbl_repository_config" columnNames="tenant_id, repository_id, key" constraintName="repository_specific_key" />
+        <addForeignKeyConstraint baseTableName="tbl_repository_config" baseColumnNames="tenant_id, key" constraintName="fk_tenant_key" referencedTableName="tbl_tenant_config" referencedColumnNames="tenant_id, key" />
```

Comment:
- This foreign key is not correct - you could have a foreign key on `tbl_global_config` but not on `tbl_tenant_config` - I'd leave it out altogether (at least until the `scope` column is added to `tbl_global_config`)
- I'm not sure about the Liquibase rules.  I thought these sorts of changes needed to go into a separate change log file, but perhaps not since there doesn't seem to be multiple files already.  We just need to make sure to test that upgrades work cleanly.